src/field_5x52_impl.h

```diff
@@ -425,6 +425,20 @@ SECP256K1_INLINE static void secp256k1_fe_mul_int(secp256k1_fe *r, int a) {
 #endif
 }
 
+SECP256K1_INLINE static void secp256k1_fe_add_int(secp256k1_fe *r, int a) {
+#ifdef VERIFY
+    secp256k1_fe_verify(r);
+    VERIFY_CHECK(a >= 0);
+    VERIFY_CHECK(a <= 0xFFFF);
```

Comment:
Is there a reason to keep it in these bounds? Sorry if it's obvious, but I don't see it.

Reply:
0x7FFF is the minimum portable MAX_INT, see https://github.com/bitcoin-core/secp256k1/pull/943#discussion_r729889981